src/Common/utils.ts

```diff
@@ -31,45 +7,50 @@ abstract class Utils {
    * This function returns true for both X-addresses and classic addresses.
    * @see https://xrpaddress.info/
    *
+   * @deprecated Please use the method provided by XrpUtils instead.
+   *
    * @param address - An address to check.
    * @returns True if the address is valid, otherwise false.
    */
   public static isValidAddress(address: string): boolean {
-    return (
-      addressCodec.isValidClassicAddress(address) ||
-      addressCodec.isValidXAddress(address)
-    )
+    return XrpUtils.isValidAddress(address)
   }
```

Comment:
I think this can be replaced with:
```
public static isValidAddress = XrpUtils.isValidAddress.bind(XrpUtils)
```

And similarly for the others that just directly call functions on XrpUtils.

Reply:
Oh! Sweet trick thank you.